src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+    const { name, age, sex, isMarried, partnerName } = person;
+
+    return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+    
+    {age && (
+      <p className="Person__age">{`I am ${age}`}</p>
+    )}
+
+    { isMarried ? (
+      <p className="Person__partner">{`${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`}</p>
+      ) : (
+        <p className="Person__partner">{`I am not married`}</p>
+      )}
```

Comment:
You can change only content instead of re-rendering different paragraphs
```suggestion
   <p className="Person__partner">
        {isMarried ? (
          `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`
        ) : (
          'I am not married'
        )}
      </p>
```